tools/mti-utils/process_intensities.xml

```diff
@@ -0,0 +1,98 @@
+<tool id="cell_intensity_processing" name="Process single-cell intensities" version="0.0.1">
```

Comment:
please add a recent profile version, this will enable a few security features :)